modules/test-generated-policies.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_module-type: PROCEDURE
+[id="test-generated-policies_{context}"]
+= Testing generated policies
+
+After you download the network policies that {product-title} generates, you can test them by applying them to your cluster.
+
+.Procedure
+. To create policies using the saved YAML file, use the following command:
++
+[source,terminal]
+----
+$ oc create -f "<generated_file>.yaml" <1>
```

Comment:
Won't hold up the PR over it, but the file name here shouldn't be in quotes. If you can consider updating in a future PR